src/cmd/cmd.cpp

```diff
@@ -155,6 +157,8 @@ void help()
     std::cout << "  --password, -p [pass]  Use [pass] as password" << std::endl;
     std::cout << "  -n                     Use netrc (5) for login" << std::endl;
     std::cout << "  --non-interactive      Do not block execution with interaction" << std::endl;
+    std::cout << "  --nonshib, -ns         Use Non Shibboleth WebDAV authentication" << std::endl;
```

Comment:
I consider myself an end-user... I only use the server installation of my employer, who again lease it as a service from SURFdrive, for which I am as far from an administrator, maintainer or developer as I could ever be. Being unable to make use of the owncloudcmd client through simple authentication options on non-customized, but possibly themed, servers at least bothers my end-user experience, especially since the nonshib davpath is not custom. I consider it relevant, else I would not have gone through the trouble of implementing it and filing a pull request.